apps/galleries_institutions/components/filter_facet/partner_filter_facet.coffee

```diff
@@ -35,9 +36,10 @@ module.exports = class PartnerFilterFacet extends Backbone.Model
     escape= (s) ->
       s.replace /[-\/\\^$*+?.()|[\]{}]/g, '\\$&'
 
+    query = AsciiFolder.fold(query)
     regex = _s.clean(escape(query)).replace(' ', '\\W* \\W*')
     substrRegex = new RegExp(regex, 'i')
-    substrRegex.test string
+    substrRegex.test AsciiFolder.fold(string)
```

Comment:
Good question -- we fold both the query term _and_ the city whose name we are searching against, so that we are doing an apples-to-apples comparison of two folded strings.

Folding the city name alone might work most of the time, but would then fail for a user who actually _does_ key in the diacritic.